modules/rostests/apitests/ntdll/NtDuplicateToken.c

```diff
@@ -134,7 +152,12 @@ DuplicateTokenAsEffective(VOID)
                               TRUE,
                               TokenPrimary,
                               &DuplicatedTokenHandle);
-    ok_hex(Status, STATUS_SUCCESS);
+    ok_ntstatus(Status, STATUS_SUCCESS);
+    if (!NT_SUCCESS(Status))
+    {
+        skip("NtDuplicateToken() failed\n");
+        goto cleanup;
+    }
```

Comment:
When the testcase has reached this path `NtDuplicateToken` is supposed to always succeed and give you the duplicated effective token so this extra code is unnecessary.